src/control/common/proto/ctl/common.pb.go

```diff
@@ -59,7 +59,7 @@ func (x ResponseStatus) String() string {
 	return proto.EnumName(ResponseStatus_name, int32(x))
 }
 func (ResponseStatus) EnumDescriptor() ([]byte, []int) {
-	return fileDescriptor_common_9d28f0bdc942251e, []int{0}
```

Comment:
It would be better to avoid committing these changes which are only for the fileDescriptor updates, as they clutter up the commit history with no benefit. I believe that it safe to ignore them by reverting them in your working tree before committing -- I have been doing this as a practice. Not worth a -1 on this PR, but something to consider for future work.

Reply:
oh alright I will keep that in mind for future patches. I still find it difficult to keep all the protobuf file changes up-to-date and in-sync with master, especially when you re-make them and it updates all the fileDescriptors as you mentioned. I will try to untrack those locally next time though